app/uk/gov/hmrc/tariffclassificationfrontend/views/forms/components/input_radiogroup.scala.html

```diff
@@ -44,10 +47,14 @@
 
         @for(option: RadioOption <- options) {
             @defining(option.value.replace(" ", "_")) { optionId: String =>
-                <div class="multiple-choice">
+                <div class="multiple-choice" @{option.dataTarget.map(dt => s"data-target=$dt").getOrElse("")}>
                     <input id="@id-@optionId" type="radio" name="@{field.name}" value="@option.value" @if(field.value.contains(option.value) && preSelect){checked="checked"}>
                     <label id="@id-@optionId-label" for="@id-@optionId">@option.label</label>
                 </div>
+                    @if(option.customHtml.isDefined) {
+                        @option.customHtml.get
```

Comment:
Think this line needs to be wrapped in a `div` with all the `panel` stuff specific to conditionally revealed content

Reply:
ok think got a solution for this, that requires less specification of stuff in client class that uses the input radio group